packages/office-ui-fabric-react/src/components/Breadcrumb/Breadcrumb.types.ts

```diff
@@ -25,6 +25,12 @@ export interface IBreadcrumbProps extends React.Props<Breadcrumb> {
    */
   className?: string;
 
+  /**
+   * Render a custom divider in place of the default chevron '>'
+   * as a custom JSX element
+   */
+  dividerAs?: JSX.Element;
```

Comment:
This should be `IComponentAs<IIconProps>` right?

Reply:
What if they want to use something other than a fabric Icon component as the divider 'icon'?

Reply:
The props we pass in are important. But it doesn't mean that they can't use something else.

For example, your span case doesn't need the props:

```
const divider = () => <span>*</span>;

<Breadcrumb dividerAs={ divider } />
```
or
```
<Breadcrumb dividerAs="*" />
```
or
```
<Breadcrumb dividerAs={ iconProps => <div>{ iconProps.iconName }</div> } />
```